### PR TITLE
Create a stats19 layer

### PR DESCRIPTION
**Pull request by dabreegster**
Frontend PR coming momentarily. Limitations are noted there. Mainly, about 155 records have coordinates that fail to transform, and I haven't looked into it yet.

Pete, please `run.sh`, upload the output to both GCS buckets, and update the manifest. `stats19.pmtiles` is 22MB for me.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Great to see this. Will take a look with ATE hat on tomorrow, or maybe before if time allows.

File: stats19/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        match collisions.get_mut(&rec.accident_index) {
+            Some(collision) => {
+                collision.severities.push(rec.casualty_severity);
+
+                if rec.casualty_type == 0 {
+                    collision.pedestrian = true;
+                } else if rec.casualty_type == 1 {
+                    collision.cyclist = true;
+                } else {
+                    // TODO These aren't all motor vehicles. What should we do with:
```
Comment (dabreegster) [inline stats19/src/main.rs:124]: Open question here

**Comment by robinlovelace-ate** [comment]
Plan to look at this in conversation with @Pete-Y-CS ~15:00. Want to join @dabreegster ?

**Comment by dabreegster** [comment]
Sure

**Review by robinlovelace-ate — APPROVED** [review]

**Review by Pete-Y-CS — APPROVED** [review]

**Pull request merged**